vcd/resource_vcd_rde.go

```diff
@@ -40,7 +40,6 @@ func resourceVcdRde() *schema.Resource {
 			"rde_type_id": {
 				Type:        schema.TypeString,
 				Required:    true,
-				ForceNew:    true,
```

Comment:
You will have to add the relevant `vcd.ResourceSchema-*` test to `skip-upgrade-tests.txt`

Reply:
Agh, I always forget about that, thanks for spotting it!

Reply:
After reviewing the tests, it seems that `vcd_resource_schema` is unable to read the `ForceNew` attribute from Terraform internals, making not possible to detect ForceNew changes in schemas. Hence it seems the test passes! So nothing to do here.